src/Yoti.Auth/AttributeConverter.cs

```diff
@@ -49,7 +49,10 @@ public static BaseAttribute ConvertToBaseAttribute(AttrpubapiV1.Attribute attrib
                     return null;
 
                 default:
-                    return null;
```

Comment:
Would there be any case when content type is none of the above.
I thought if photo doesn't know the content type, it will be set to Undefined. 
I am not sure we need this case here.

Reply:
If a new content type is added, e.g. `Bitmap = 6`, then if we haven't updated the SDKs then I believe it would go down this route. 

If I remember correctly, `Undefined` content types are only used if there's been an error. @bucky-boy do you remember? 

In any case, we might not need it (or we might be very unlikely to need it), but is there any harm in leaving it here?

Reply:
👍